### PR TITLE
Added Get method to hideUI to show ui

### PR DESCRIPTION
**Pull request by puzzledShark**

### CODE REVIEW

File: lib/hideUI.js [hunk 1]
```diff
@@ -12,10 +12,23 @@ define(function (require, exports, module) {
...
+			removeRightSideToolBar();
+			removeTitleBar();
+		}
+	}
+
+	/**
+	 * This function parses the current url for brackets, and looks for Get Methods
+	 */
+	function _parseParams(sParam) {
+		var sPageURL = window.location.search.substring(1);
```
Comment (humphd) [inline lib/hideUI.js]: Instead of this, you can use the Brackets API to do this, see https://github.com/humphd/brackets-browser-livedev/pull/59

Comment (humphd) [inline lib/hideUI.js]: wrong link, sorry https://github.com/adobe/brackets/blob/master/src/brackets.js#L148-L152

Comment (humphd) [inline lib/hideUI.js]: So I'd rename this to `function shouldHideUI()` and have it return `true` or `false` depending on whether that param is set to `1` or not.

Comment (humphd) [inline lib/hideUI.js]: Also, you need to document how to use this above the function, indicating that `?ui=1` will leave UI in tact.

File: lib/hideUI.js [hunk 2]
```diff
@@ -4,18 +4,39 @@
...
+	/**
+	 * This function parses brackets URL, and looks for the GET parameter "ui"
+	 * if ui is set to 1, then the UI is shown
+	 * if ui is set to any other number then ui is not shown
+	 * if ui is not set, the ui is not shown
+	 */
+	function shouldHideUI() {
+		var params = new UrlParams();
+		params.parse();
+		params["_store"]["ui"];
```
Comment (humphd) [inline lib/hideUI.js]: Use `get` instead, and simplify this to something like: 

``` js
var params = new UrlParams();
params.parse();
return params.get("ui") === "1";
```

Comment (puzzledShark) [inline lib/hideUI.js]: whoops, that params["_store"]["ui"]; is from my testing, forgot to take it out

File: lib/hideUI.js [hunk 3]
```diff
@@ -4,18 +4,33 @@
...
+			removeLeftSideToolBar();
+			removeRightSideToolBar();
+			removeTitleBar();
+		}
+	}
+
+	/**
+	 * This function parses brackets URL, and looks for the GET parameter "ui"
+	 * if ui is set to 1, then the UI is shown
+	 * if ui is set to any other number then ui is not shown
```
Comment (humphd) [inline lib/hideUI.js]: You can get rid of the last 2 lines of this comment block.

**Comment by humphd** [comment]
One nit, then this is good.

**Comment by humphd** [comment]
For some reason you're using 8-space indents, and we use 4.  Fix that too, please.

**Comment by humphd** [comment]
Next time we won't do whitespace changes in a code change (use a separate bug for global whitespace fixups) but this is OK for now.  r+

**Pull request merged**